browser-test/src/support/applicant_questions.ts

```diff
@@ -709,6 +709,7 @@ export class ApplicantQuestions {
     wantUpsell: boolean,
     wantTrustedIntermediary: boolean,
     wantEligiblePrograms: string[],
+    northStarEnabled = false,
```

Comment:
**suggestion:** We can remove this flag, since we don't use it and we have the northstar-specific version below.

Reply:
Done.